lib/internal/per_context/primordials.js

```diff
@@ -197,5 +197,16 @@ primordials.SafeWeakSet = makeSafe(
   copyPrototype(original.prototype, primordials, `${name}Prototype`);
 });
 
+// Create copies of WebAssembly objects.
+// Refs: https://webassembly.github.io/spec/js-api/index.html#idl-index
+// Refs: https://heycam.github.io/webidl/#idl-namespaces
+Object.entries(WebAssembly).forEach(([ctorName, original]) => {
+  const name = `WebAssembly${ctorName[0].toUpperCase() + ctorName.slice(1)}`;
+  primordials[name] = original;
+
+  copyPropsRenamed(original, primordials, name);
+  copyPrototype(original.prototype, primordials, `${name}Prototype`);
+});
+
```

Comment:
Shouldn't this go along `JSON`, `Math`, and `Reflect`?

Reply:
No, because the `WebAssembly` namespace contains constructors.